build/entitlements.mac.plist

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>com.apple.security.device.microphone</key>
+	<true/>
+	<key>com.apple.security.device.camera</key>
+	<true/>
+	<key>com.apple.security.device.audio-input</key>
```

Comment:
Just curiosity. What covers audio-input and why it is needed? Isn't audio input the same as microphone?

Reply:
I wish I could tell, from my tests I wasn't being able to share the camera until I added this one. It's description is that it is used to record audio from the mic, so apparently it is redundant but it didn't work without it 🤷‍♂️. My guess is that jitsi allows to change its source (when you have more than one), but it is just a guess.